src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -67,7 +67,7 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
                 .ConfigureLogging((context, loggingBuilder) =>
                 {
                     loggingBuilder.ClearProviders();
-
+                    loggingBuilder.AddConsole();
```

Comment:
Do you mind explaining what this change does?

Reply:
good catch...unintenational. This is useful for local debugging to see logs from worker processes.